libs/portals/admin/petition/src/components/ExportList/index.tsx

```diff
@@ -22,24 +32,56 @@ interface Props {
 export const getCSV = async (data: any[], fileName: string) => {
   const name = `${fileName}`
   const dataArray = data.map((item: any) => [
-    item.created ?? '',
+    formatDate(item.created) ?? '',
     item.meta.fullName ?? '',
+    item.meta.locality ?? '',
   ])
 
-  await downloadCSV(name, ['Dagsetning', 'Nafn'], dataArray)
+  await downloadCSV(name, ['Dagsetning', 'Nafn', 'Sveitarfélag'], dataArray)
 }
 
-const ExportList: FC<Props> = ({ onGetCSV, dropdownItems = [] }) => {
+export const ExportList: FC<Props> = ({
+  petition,
+  petitionSigners,
+  petitionId,
+  onGetCSV,
+  dropdownItems = [],
+}) => {
   const { formatMessage } = useLocale()
+
+  const [document] = usePDF({
+    document: (
+      <MyPdfDocument petition={petition} petitionSigners={petitionSigners} />
+    ),
+  })
+
+  if (document.error) {
+    console.warn(document.error)
+  }
+
   return (
     <Box className={styles.buttonWrapper}>
       <DropdownMenu
         icon="download"
+        iconType="outline"
         menuLabel={formatMessage(m.downloadPetitions)}
         items={[
+          {
+            title: formatMessage(m.asPdf),
+            render: () => (
+              <a
```

Comment:
Should we use something like `<Link>` or is that not applicable/possible in this case seeing that this is some special render for the pdf?

Reply:
sí